src/web/methods/index.ts

```diff
@@ -0,0 +1,6 @@
+import { click } from './interactions';
```

Comment:
It's unmaintainable. We shouldn't need to import methods one by one, but import all of them at once (import * as intercations from './interactions'.

Another point, it should be named kakMethods, but probably just methods.

Instead of exporting single method it should export all methods from single group, under single key.